src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println(new StringBuilder()
+                .append(getName())
+                .append(" Excavator started its work")
+                .toString());
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println(new StringBuilder()

```

Comment:
Usage of StringBuilder for simple string concatenation is unnecessary and can be replaced with a simple '+' operation for cleaner code.